crm_lead_stage_monitoring/crm.py

```diff
@@ -35,9 +35,10 @@ def write(self, cr, uid, ids, vals, context=None):
             vals['last_activity_stage'] = fields.datetime.now()
         return super(CrmLead, self).write(cr, uid, ids, vals, context=context)
 
-    def copy(self, cr, uid, id, default=None, context=None):
+    def copy(self, cr, uid, rec_id, default=None, context=None):
```

Comment:
It seams that could have some side effect with old/new API compatibility when guessing how it needs to be decorated:
https://github.com/odoo/odoo/blob/8.0/openerp/api.py#L641
So I think we can set `rec_id` but we need to specify its decorator:
`@api.cr_uid_id_context`

Or we can revert to `id`